projects/samples/contests/robocup/controllers/model_verifier/README.md

```diff
@@ -0,0 +1,65 @@
+# Model Verifier
+
+This folder contains the `model_verifier.py` controller which spawns automatically a robot and automatically searches
+for infringements to the rules or suspicious elements.
+
+Keep in mind that this controller only checks **part of the rules**, therefore it's possible that a robot is invalid
+although it did not trigger errors nor warning during the analysis.
+
+## Dependencies
+
+Additionally to `Webots`, you will need the following modules to run the `model_verifier.py` controller:
+
+`pip3 install trimesh transforms3d numpy pandas tabulate`
```

Comment:
This should be moved to a `requirements.txt` file.

Reply:
That's  done.